### PR TITLE
Creating the project release process documents

### PR DESCRIPTION
**Pull request by mdubya66**
This is the beginning of these. Please review and we will revise over time.

### CODE REVIEW

File: project/RELEASE-DETAILS.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* Follow the [RELEASE-VERSIONING.md](RELEASE-VERSIONING.md) for choosing the new version.
+* Gather the Release Notes file.
+* Go to https://github.com/vmware/vic/releases and click on Draft a new release.
+* Add the tag version that meets the [RELEASE-VERSIONING.md](RELEASE-VERSIONING.md) requirements for this release.
+* Title the release "vSphere Integrated Containers Version X.Y.Z" where X.Y.Z meets the versioning requirements. 
+* Paste the release note file contents into the Write field and preview.
+* If this is release is non-production ready select the "This is a pre-release" box.
+
+
+## Create the VIC binary package
```
Comment (dougm) [inline project/RELEASE-DETAILS.md]: Should add a TODO here and/or link to an issue, the upload from drone to bintray can all be automated via drone.yml.
The only manual part should be creating the git tag - we need to add the git tag steps to this doc in any case.

File: project/RELEASE-PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# VIC Release Process
+
+This document describes how the VIC project is released.
+
+## Release cycle
+
+VIC is developed with a feature driven model. Requirements are broken down into
+features and then sized to ensure they will fit into our monthly releases.  Any
+features that do not are then reworked until they will fit. Any features under
+active development that do not make the montly release will be pushed into the
```
Comment (dougm) [inline project/RELEASE-PROCESS.md]: s/montly/monthly/

File: project/RELEASE-VERSIONING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# VIC Release Versioning
+
+This document provides examples of VIC release versions.
+
+The VIC project follows the general vX.Y.Z versioning scheme. Where X is Major, Y is
```
Comment (dougm) [inline project/RELEASE-VERSIONING.md]: Maybe prune this section a bit and just link to http://semver.org/

Comment (dougm) [inline project/RELEASE-VERSIONING.md]: Just realized this is 3 separate files, not sections.. any reason not to have these in a single file?

File: project/RELEASE-DETAILS.md (hunk 1 above)
Comment (mdubya66) [inline project/RELEASE-DETAILS.md]: TODO added

File: project/RELEASE-PROCESS.md (hunk 2 above)
Comment (mdubya66) [inline project/RELEASE-PROCESS.md]: fixed typo

File: project/RELEASE-VERSIONING.md (hunk 3 above)
Comment (mdubya66) [inline project/RELEASE-VERSIONING.md]: Updated VERSION to reference semver.org. Merged files. We can split them later if it gets to large.

**Comment by dougm** [comment]
👍  for a single doc.  Need to remove some of the old links now.  Also check the markup, version list for example is not rendering as a list: https://github.com/mdubya66/vic/blob/release/project/RELEASE-PROCESS.md

**Comment by andrewtchin** [comment]
lgtm

**Comment by dougm** [comment]
@mdubya66 for next time: https://github.com/vmware/vic/blob/master/CONTRIBUTING.md#updating-pull-requests

We could also enable this as an option: https://github.com/blog/2141-squash-your-commits
(need someone with repo admin permission to do it)

**Comment by mdubya66** [comment]
Got it. Can we add instructions on how to get the <commit> id?


--
Matt Williamson
mwilliamson@vmware.com<mailto:mwilliamson@vmware.com>
(W) 512 813 3455
(C) 512 496 1788

From: Doug MacEachern
Reply-To: vmware/vic
Date: Monday, May 2, 2016 at 12:43 PM
To: vmware/vic
Cc: Matt Williamson, Mention
Subject: Re: [vmware/vic] Creating the project release process documents (#603)


@mdubya66<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mdubya66&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=_HNG7MsT3qaLn7_f1XRJR0MicqkG6oi-42eFTYWaU4E&m=UenEMVTTijNlUclpVIsDLw6OKVV8_RqDInzDkDrYIM4&s=X4EvbtByf_BillXEl9n6SPHlXuw0rnfIIaUzM8C2v6A&e=> for next time: https://github.com/vmware/vic/blob/master/CONTRIBUTING.md#updating-pull-requests<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_vic_blob_master_CONTRIBUTING.md-23updating-2Dpull-2Drequests&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=_HNG7MsT3qaLn7_f1XRJR0MicqkG6oi-42eFTYWaU4E&m=UenEMVTTijNlUclpVIsDLw6OKVV8_RqDInzDkDrYIM4&s=Ff89Eo2kyWjr4CdKLSI98NcfcfHMBbnNJoAprVVq-n0&e=>

We could also enable this as an option: https://github.com/blog/2141-squash-your-commits<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_blog_2141-2Dsquash-2Dyour-2Dcommits&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=_HNG7MsT3qaLn7_f1XRJR0MicqkG6oi-42eFTYWaU4E&m=UenEMVTTijNlUclpVIsDLw6OKVV8_RqDInzDkDrYIM4&s=WAFpFS0vUpZVBhb-lNrdQgHNe-2yuX1cet0pY__iCZY&e=>
(need someone with repo admin permission to do it)

**Comment by dougm** [comment]
I was shown that the PR squash option is enabled already on our repo.

You can use 'git log' for example to see commit ids.  I'll update the doc in a bit.

**Pull request merged**